repository/src/main/java/org/alfresco/repo/security/authority/AuthorityDAOImpl.java

```diff
@@ -1431,7 +1442,31 @@ public void setAuthorityDisplayName(String authorityName, String authorityDispla
             return;
         }
         nodeService.setProperty(ref, ContentModel.PROP_AUTHORITY_DISPLAY_NAME, authorityDisplayName);
+    }
 
+    @Override
+    public Pair<String, String> getAuthorityDisplayNameAndDescription(String authorityName)

```

Comment:
I think you could call through to `getAuthorityDisplayName` to reduce the duplication here.

Reply:
Okay I was actually considering doing so but then I think it would mean the second `getAuthorityOrNull` call in `getAuthorityDisplayName` would it be fine for us? I mean as far as I checked it uses `authorityLookupCache` so it might be fine what do you think?